content/blog/grpc-web-interceptor.md

```diff
@@ -209,6 +163,31 @@ TestStreamInterceptor.prototype.intercept = function(request, invoker) {
 };
 ```
 
+## Binding interceptors
+
+By passing an array of inteceptor instances using an appropriate option key,
+you can bind interceptors to a client when the client is instantiated:
+
+```js
+const promiseClient = new FooServicePromiseClient(
+    host, creds, {'unaryInterceptors': [interceptor1, interceptor2, interceptor3]});
+
+const client = new FooServiceClient(
+    host, creds, {'streamInterceptors': [interceptor1, interceptor2, interceptor3]});
+```
+
+{{< note >}}
+  Interceptors are executed in reverse order for request processing, and in order for response processing:
+
+  ```nocode
+  Request  --> interceptor3  --> interceptor2  --> interceptor1 --> Intercepted --> Server
+                                                                    request           |
+                                                                                      |
+  Intercepted <-- interceptor3 <-- interceptor2 <-- interceptor1 <--   Response  <-- Server
+  response
+  ```
+{{< /note >}}
+
 ### Advanced example: RetryInterceptor
```

Comment:
As an introduction to grpc-web interceptors, I'd drop this "advance example".

IMHO, it might be more useful be able to include links to running examples.

Reply:
We haven't ported all of our examples to Github yet. The only one is here. We may add more examples to this file or under this directory in the future. 
https://github.com/grpc/grpc-web/blob/cc6cdfb428f21dd70b9e714f0dacc0ec0745d06b/net/grpc/gateway/examples/echo/commonjs-example/client.js

Reply:
So, shall we drop this example, or do you prefer to keep it?

Reply:
You can drop this example. I think we can add it somewhere under https://github.com/grpc/grpc-web/blob/cc6cdfb428f21dd70b9e714f0dacc0ec0745d06b/net/grpc/gateway/examples/echo/commonjs-example/

Reply:
> You can drop this example

Done